src/store/features/entities/entitiesTypes.ts

```diff
@@ -99,7 +99,7 @@ export type Trigger = {
   width: number;
   height: number;
   script: string[];
-  leaveScript: string[];
+  leaveScript?: string[]; // Optional: Old versions of GBS don't have this script
```

Comment:
@chrismaltby let me know if you prefer a different way to deal with this case. Maybe a migration script?

Reply:
Yeah I think a migration script so we can expect all triggers to at least have an empty array would be preferable. With the latest v3beta branch you no longer need an "EVENT_END" event in the array for it work so a migration to just add `leaveScript: []` to every trigger should be enough :-)

Reply:
v3alpha even, you know what I mean :-)